src/CONST.js

```diff
@@ -4,6 +4,7 @@ const NEW_EXPENSIFY_URL = 'https://new.expensify.com';
 const CONST = {
     // 50 megabytes in bytes
     API_MAX_ATTACHMENT_SIZE: 52428800,
+    AVATAR_MAX_ATTACHMENT_SIZE_MB: 3,
```

Comment:
Could you please change it to Bytes? Most of the time arithmetic operations will be in bytes.